integration-test/trino-it/launch.sh

```diff
@@ -29,15 +32,34 @@ echo "The docker compose log is: $LOG_PATH"
 
 nohup docker compose logs -f  -t >> $LOG_PATH &
 
-max_attempts=600
+max_attempts=0
 
-for ((i = 0; i < max_attempts; i++)); do
+while true; do
     docker compose exec -T trino trino --execute "SELECT 1" >/dev/null 2>&1 && {
-        echo "All docker compose service is now available."
-        exit 0
+        break;
     }
+
+    num_container=$(docker ps --format '{{.Names}}' | grep trino-ci | wc -l)
+    if [ "$num_container" -lt 4 ]; then
+        echo "ERROR: Trino-ci containers start failed."
+        exit 0
+    fi
+
     sleep 1
+
+    if [ "$max_attempts" -ge 600 ]; then 
+        echo "ERROR: Trino service did not start within the specified time."
+        exit 1
+    fi
+    ((count++))
 done
 
-echo "Trino service did not start within the specified time."
-exit 1
+
+echo "All docker compose service is now available."
+
+docker exec trino-ci-hive chown -R `id -u`:`id -g` /tmp/root
+docker exec trino-ci-hive chown -R `id -u`:`id -g` /usr/local/hadoop/logs
+ls -l ../build/trino-ci-container-log
+ls -l ../build/trino-ci-container-log/hive
+ls -l ../build/trino-ci-container-log/hdfs
```

Comment:
What's the usage of the ls command?

Reply:
For debug, When checking the error, we can see the details of the permissions.